code/_onclick/click.dm

```diff
@@ -116,6 +116,7 @@
 		return
 
 	if(in_throw_mode)
+		SEND_SIGNAL(src, COMSIG_MOB_THROW, A, params)

```

Comment:
Put this signal in the throw_item proc

Reply:
it already has a signal that's sent from the item being thrown